src/qt/addressbookpage.cpp

```diff
@@ -16,6 +16,7 @@
 #include <qt/platformstyle.h>
 
 #include <QIcon>
+#include <QLatin1String>
```

Comment:
Uhm, I'm a bit lost, why do we have QLatin1String is included a lot in this PR?

Reply:
Added the header where it is used but header is missing.